docs/reference/api-docs.asciidoc

```diff
@@ -265,6 +265,8 @@ ConfigSource references configuration settings.
 |===
 
 
+
+
```

Comment:
nit: remove?

Reply:
I don't know where it comes from, it auto-generates this way :|
I see other places in this file with a lot of empty lines.
@charith-elastic any idea?

Reply:
They don't get rendered IIRC so I'm not sure it's worth spending too much time on